src/modules/ProductLicense.rb

```diff
@@ -1711,6 +1711,7 @@ def product_license(id, tmpdir)
       product_names.each do |product_name|
         locales = Pkg.PrdLicenseLocales(product_name)
         log.info("License locales for product #{product_name.inspect}: #{locales.inspect}")
+        next unless locales
```

Comment:
another option could be simple `locales ||= []` which will behave same.

But in either case please document why this check is needed and ideally also with bug report reference.